client/src/styles/legacy/_style.scss

```diff
@@ -688,7 +688,10 @@ html, body {
 
 .validationerror .panel--padded .ui-tabs-nav {
   margin-top: -$panel-padding-y;
+}
 
+.validationerror .panel--padded .ui-tabs-nav,
+.validationerror .toolbar--north .ui-tabs-nav {
   .tab-attention {
     color: $state-draft;
     position: absolute;
```

Comment:
`.panel--padded` isn't present in `GridField` edit forms, and would be at the wrong level even if it was present. This is needed mostly to make sure the tab icon has the correct colour.